tensorflow/compiler/tf2tensorrt/BUILD

```diff
@@ -507,7 +507,7 @@ tf_cuda_library(
     ] + if_tensorrt([":tensorrt_lib"]),
 )
 
-tf_cc_test(
```

Comment:
This is tf_cc_test import from line 10 useless. Need to remove that from the import.

Reply:
Fixed